CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# Contributing
+
+## Questions
+
+If you are having difficulties using the APIs or have a question about the IBM Cloud and Services,
+please ask a question on [dW Answers][dw] or [Stack Overflow][stackoverflow].
+
+## Issues
+
+If you encounter an issue with the Python SDK, you are welcome to submit a [bug report](https://github.com/IBM/python-sdk-core/issues).
+Before that, please search for similar issues. It's possible somebody has encountered this issue already.
+
+## Pull Requests
+
+If you want to contribute to the repository, here's a quick guide:
+
+1. Fork the repository
+1. Install `virtualenv` and `tox`
+1. Develop and test your code changes with [pytest].
+    * Respect the original code [style guide][styleguide].
+    * Only use spaces for indentation.
+    * Create minimal diffs - disable on save actions like reformat source code or organize imports. If you feel the source code should be reformatted create a separate PR for this change.
+    * Check for unnecessary whitespace with `git diff --check` before committing.
+    * Make sure your code supports Python 2.7, 3.4, 3.5 and 3.6. You can use `pyenv` and `tox` for this
+1. Make the test pass
+1. Commit your changes
+* Commits should follow the [Angular commit message guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines). This is because our release tool uses this format for determining release versions and generating changelogs. To make this easier, we recommend using the [Commitizen CLI](https://github.com/commitizen/cz-cli) with the `cz-conventional-changelog` adapter.
+1. Push to your fork and submit a pull request to the `dev` branch
+
+## Running the tests
+
+You probably want to set up a [virtualenv].
+
+1. Clone this repository:
+    ```sh
+    git clone https://github.com/watson-developer-cloud/python-sdk.git
```

Comment:
https://github.com/watson-developer-cloud/python-sdk.git should be https://github.com/watson-developer-cloud/python-sdk-core.git

Reply:
Actually the git URL should be `https://github.com/IBM/python-sdk-core.git`